src/BffElectionRunner.php

```diff
@@ -0,0 +1,115 @@
+<?php
+namespace PivotLibre\Tideman;
+
+class BffElectionRunner
+{
+    private $ballotParser;
+    private $nBallotParser;
+    private $candidateRankingSerializer;
+    private $tieBreaker;
+
+    public function __construct()
+    {
+        $this->ballotParser = new BallotParser();
+        $this->nBallotParser = new NBallotParser();
+        $this->candidateRankingSerializer = new CandidateRankingSerializer();
+    }
+
+    /**
+     * @param string $bffTieBreaker a tie breaking ballot formatted according to BFF.
+     */
+    public function setTieBreaker(string $bffTieBreaker) : void
+    {
+        $tieBreaker = $this->ballotParser->parse($bffTieBreaker);
+        $this->tieBreaker = $tieBreaker;
+    }
+
+    /**
+     * @param string newline-delimited ballots formatted according to BFF
+     * @return string BFF-encoded result
+     */
+    public function run(string $bffBallots) : string
+    {
+        $bffBallotStrings = $this->splitOnNewLine($bffBallots);
+        $ballots = $this->parseMultipleBallotStrings(...$bffBallotStrings);
+        $result = $this->runHelper(...$ballots);
+        return $result;
+    }
+
+    /**
+     * @param string ...$bffBallots one BFF ballot per string
+     * @return string BFF-encoded result
+     */
+    public function runAll(string ...$bffBallots) : string
```

Comment:
The following method is useful to a user of this class that has already split their ballots up by newline (or other delim).